### PR TITLE
fix: don't show crd link, when source is Push

### LINKED ISSUE #2184: In settings > topology, If source is Push CRD link should not be shown (it crashes) and allow option to delete
Example in last line: https://app.flanksource.com/settings/topologies/204c57e8-2226-4f2b-a53d-61c88f7ab580

### PR DESCRIPTION
**Pull request by mainawycliffe**
Fixes #2184

### CODE REVIEW

File: src/components/Settings/CRDSource.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function CRDSource({
   source,
   showMinimal = false
 }: CRDSourceProps) {
-  if (!source || source === "UI") {
+  if (!source || source === "UI" || source.toLowerCase() === "push") {
```
Comment (yashmehrotra) [inline src/components/Settings/CRDSource.tsx:19]: I think we should inverse this logic, ie

```
if source != "crd" {return null}

return (
  <comp>
)
```

**Comment by moshloop** [comment]
@mainawycliffe  There is no save/update options for UI topologies now ?

**Comment by moshloop** [comment]
Can we change the link for agent to be `RiRobot2Fill Agent`

**Comment by moshloop** [comment]
@mainawycliffe  - It is wrapping, has an underline and it says `link` instead of `Agent`

**Pull request merged**